src/script/script.h

```diff
@@ -507,14 +507,18 @@ class CScript : public CScriptBase
     {
         if (opcode == OP_0)
             return 0;
+        if (opcode == OP_1NEGATE)
+            return -1;
         assert(opcode >= OP_1 && opcode <= OP_16);
```

Comment:
Non-blocking comment: Maybe move the `assert` line at the beginning of this member function so it is even more similar to its `EncodeOP_N` counterpart.